### ISSUE TITLE
Missing ASD.npz 

### ISSUE THREAD

**Issue by Coderx7** [Mahmoud-Elbattah/Predicting_ASD#3]
Hi @Mahmoud-Elbattah , 
First of all Thank you very much for sharing your work. 
May I ask you to kindly share the `ASD.npz` as well or the script you used to create it? 
I'm trying to run your ipython notebooks and hopefully work on the dataset, but this file is missing and I also couldn't find any scripts in the repo to build this. 
Really appreciate your kind help 
Best Regards

**Comment by Mahmoud-Elbattah** [comment]
Hello,
Thanks for your interest in our work.
The file is too large to upload on GitHub.
So, I've added the script: imgsNPZ.py
Regards,

**Comment by Coderx7** [comment]
Thanks a lot for the quick reply. 
I noticed the [cropping.py](https://github.com/Mahmoud-Elbattah/Predicting_ASD/blob/master/cropping.py#L24) uses a different dataset it seems. the images are named differently(i.e. there is no 'neg`/`pos` in file names) and are in a different extension as the ones you kindly previously provided [here](https://figshare.com/s/5d4f93395cc49d01e2bd). 
Could you kindly also provide the updated dataset? or in case they are the same, update the script to reflect this?

**Comment by Mahmoud-Elbattah** [comment]
Hello,
It is the same dataset that we are sharing on Figshare.
We  just renamed the inner folders (i.e. TS-> pos, TC->neg).
Regards,

**Comment by Coderx7** [comment]
Thanks a lot greatly appreciate it.

**Comment by Coderx7** [comment]
@Mahmoud-Elbattah Hi, 
I noticed the preprocessings that is currently available in this repo, seems to be different from the one that was actually used in the `DNN Model(Keras).py`, as the dimensions do not match. `Data%20Preprocessing.py` creates an array of shape : my_data: `(8451, 10001)`, while the `DNN Model(keras).py` expects this to be 40K.
Am I missing something here? would greatly appreciate if you could kindly have a look and hopefully update this as well.
Thanks a lot in advance

**Comment by Mahmoud-Elbattah** [comment]
Hello,
This depends on the image dimensions that you'd apply.
In one case, we resized the images to 100x100 (i.e. 10k), 
In the DNN models, we used larger dimensions.

You can find more details in our paper.
https://www.researchgate.net/publication/331784416_Learning_to_Predict_Autism_Spectrum_Disorder_based_on_the_Visual_Patterns_of_Eye-tracking_Scanpaths

And you can definitely follow different pre-processing procedures.

Regards

**Comment by Coderx7** [comment]
Thanks a lot really appreciate it. 
Yes, of course, but I was trying to first replicate the base line, and I presumed the notebooks are configured in a way that would simply just allow for that. anyway everything is great and thank you again for your kind help. 
Have a wonderful day

**Comment by Coderx7** [comment]
@Mahmoud-Elbattah Hi, 
I was reading the paper, and noted the table 3, where you specified the neural networks and the experiments you ran. 
I tried to implement that, but I cant seem to be getting any where more than 80. 
Could you also kindly share the architecture or the script you used to get that 92% accuracy? 
It would greatly be appreciated as I'm trying to not only replicate your results, but also changing different configurations and see how they affect the end result. 
I know I can just go about anything, but I need to be able to replicate your results first so I can have a standard to check my configs against yours. 
Again thank you again for kind help

**Comment by Mahmoud-Elbattah** [comment]
Hello,
All the scripts are already shared on the repo.
One important point to consider is that the experiments reported in our paper did not split the data by participants. 
In other words, (different) images related to one participant could have existed in the train and test sets.
You’d achieve like 10% lower accuracy if you split the dataset based on participants.

Perhaps I could help you better if I get a clear idea about what you’re trying to achieve.
Please feel free to contact me if you’d like to discuss this further.
mahmoud.elbattah@u-piacrdie.fr

Regards,
Mahmoud

**Comment by Coderx7** [comment]
Thanks a lot really appreciate your kind reply. I'll prepare something for this and email you when the codes are ready so you can also see first hand how things are implemented from preprocessing (which by the way is what I have found here) + the models itself.
Have a wonderful day

**Comment by Coderx7** [comment]
Is the email address correct? becasue the email is not delivered and this is what I get : 

![image](https://user-images.githubusercontent.com/5382892/99111186-e80db500-2600-11eb-9cdd-71e9082a1ad8.png)

**Comment by Mahmoud-Elbattah** [comment]
Hello,
Apologies, it is:
mahmoud.elbattah@u-picardie.fr
